CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 - Updated es5-shim npm module to version `4.5.9` from `4.5.7`.
 - Updated gulp-rename npm module to version `1.2.2` from `1.1.0`.
 - Replaces deprecated gulp-cssmin with gulp-clean-css.
+- Updated admin page sidefoot label and icon to read 'Related content'
```

Comment:
Could probably throw in a "Related links" here to indicate _which_ label and icon have been updated ;)